packages/compass-preferences-model/index.d.ts

```diff
@@ -1,15 +1,29 @@
-declare class CompassPreferencesModel {
-  fetch: () => void;
-  save: () => void;
-  set: (
-    key: string,
-    value: unknown
-  ) => void;
-};
-export default CompassPreferencesModel;
-
 export enum THEMES {
   DARK = 'DARK',
   LIGHT = 'LIGHT',
   OS_THEME = 'OS_THEME'
-};
+}
+
+export type UserPreferences = {
+  showedNetworkOptIn: boolean; // Has the settings dialog has been shown before.
+  autoUpdates: boolean;
+  enableMaps: boolean;
+  trackErrors: boolean;
+  trackUsageStatistics: boolean;
+  enableFeedbackPanel: boolean;
+  theme: THEMES;
+}
+
+declare class CompassPreferencesModel {
+  fetchPreferences: () => Promise<void>;
+  savePreferences: (preferences: any) => Promise<void>;
+  getPreferenceValue: any;
```

Comment:
Just leaving a note here to property type these before it’s merged

Reply:
Yeah, there are many any :) Don't worry, I will replace them with proper types before converting PR from draft to open.